articles/custom-os-settings.md

```diff
@@ -24,26 +24,45 @@ Fleet UI:
-2. Run `sudo profiles renew -type enrollment` on automatically enrolled Mac.
+If a host is automatically enrolled (via [ADE](https://support.apple.com/en-us/102300)), user-scoped profiles are delivered to the user that was created during first time setup. For hosts that enrolled and turned on MDM manually, user-scoped profiles are delivered to the user that installed Fleet's agent (fleetd).
 
-> Fleet will soon improve this and automatically enable the user channel for all macOS hosts. Check out the [issue](https://github.com/fleetdm/fleet/issues/30043).
+How to deliver user-scoped configuration profiles:
+1. If you use iMazing Profile Creator, open your configuration profile in iMazing, select the **General** tab and update the **Payoad Scope** to **User**.
+2. If you edit your configuration profiles in a text editor, open the configuraiton profile in your text editor, find or add the `PayloadScope` key, and set the value to `User`. Here's an example `.mobileconfig` snippet:
 
-Support for declaration (DDM) profiles is coming soon.
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	...
+	<key>PayloadScope</key>
+	<string>User</string>
+</dict>
+</plist>
+```
+
+#### Upgrading to 4.71.0
+
+When upgrading to 4.71.0, here's how to prepare your already enrolled hosts for user-scoped configuration profiles:
+1. If the host automatically enrolled to Fleet (via ADE), you don't need to take action. Fleet added support for the user-scoped configuration profiles on these hosts.
+2. To deliver user-scoped profiles to hosts that manually enrolled and turned on MDM, first turn off MDM and ask end user to [turn on MDM](https://fleetdm.com/guides/mdm-migration#migrate-hosts:~:text=If%20the%20host%20is%20not%20assigned%20to%20Fleet%20in%20ABM%20(manual%20enrollment)%2C%20the%20end%20user%20will%20be%20given%20the%20option%20to%20download%20the%20MDM%20enrollment%20profile%20on%20their%20My%20device%20page.) through the **My device** page.
+
+When upgrading to Fleet 4.71.0, here's how to update configuration profiles that are already installed on hosts so that they're delivered to the user scope:
 
-Existing profiles with `PayloadScope` set to`User` won’t update automatically. These are delivered to the device channel and will remain there until you take action.
+1. Check for profiles with `PayloadScope` set to `User`. Already deployed profiles with `PayloadScope` set to `User` won’t be re-installed on hosts automatically.
+2. To change them to the user-scope and re-install them, update the `PayloadIdentifier` and re-add the profile to Fleet. This will uninstall the device-scope profile and install the profile in the user scope.
```

Comment:
"This will uninstall the device-scope profile and install the profile in the user scope."

@marko-lisica @JordanMontgomery this is true right?

Reply:
Ah, I see. GitOps it's just an edit but in the UI it's an edit, re-add, remove.

I updated the copy. Let me know what y'all think. Please feel free to tweak it yourselves too.